qualtricssurvey/private/edit.html

```diff
@@ -24,6 +23,110 @@
               of the page.
           </span>
       </li>
+      
+      <li class="field comp-setting-entry is-set">
+          <div class="wrapper-comp-setting">
+              <label
+                  class="label setting-label"
+                  for="xblock_qualtricssurvey_survey_id"
+              >
+                  Survey ID
+            </label>
+            <input
+                class="input setting-input"
+                id="xblock_qualtricssurvey_survey_id"
+                value="{self.survey_id}"
+                type="text"
+                tabindex="2"
+            >
+          </div>
+          <span
+              class="tip setting-help"
+          >
+              This is the ID that Qualtrics uses for the survey,
+               usually an 8 digit number.
+          </span>
+      </li>
+      
+      <li class="field comp-setting-entry is-set">
+          <div class="wrapper-comp-setting">
+              <label
+                  class="label setting-label"
+                  for="xblock_qualtricssurvey_your_university"
+              >
+                  Your university
+            </label>
+            <input
+                class="input setting-input"
+                id="xblock_qualtricssurvey_your_university"
+                value="{self.your_university}"
+                type="text"
+                tabindex="3"
+            >
+          </div>
+          <span
+              class="tip setting-help"
+          >
+              This is the name of your university.
+          </span>
+      </li>
+      
+      <li class="field comp-setting-entry is-set">
+          <div class="wrapper-comp-setting">
+              <label
+                  class="label setting-label"
+                  for="xblock_qualtricssurvey_link_text"
+              >
+                  Link text
+            </label>
+            <input
+                class="input setting-input"
+                id="xblock_qualtricssurvey_link_text"
+                value="{self.link_text}"
+                type="text"
+                tabindex="4"
+            >
+          </div>
+          <span
+              class="tip setting-help"
+          >
+              This is the text that will link to your survey.
+          </span>
+      </li>
+      
+      <li class="field comp-setting-entry is-set">
+          <div class="wrapper-comp-setting">
+              <label
+                  class="label setting-label"
+                  for="xblock_qualtricssurvey_param_name"
+              >
+                  Parameter Name for UserID
+            </label>
+            <input
+                class="input setting-input"
+                id="xblock_qualtricssurvey_param_name"
+                value="{self.param_name}"
+                type="text"
+                tabindex="5"
+            >
+          </div>
+          <span
+              class="tip setting-help"
+          >
+              This is the name for the User ID parameter in the url.
```

Comment:
I see that you have removed the text from the instance variable to this field. Is it possible to leave this text as the 'help' field for the instance variable in param_field in qualtricssurvey.py, and then access it here? That way this text is used to help developers as well as users. Is it possible to access the help field from here @stvstnfrd ?